docs/source/notes/libtorch_stable_abi.md

```diff
@@ -30,4 +30,21 @@ This note will eventually contain more details on how to use the APIs in torch/c
 | ? | ? | c10::SymBool | SymBool |
 | ? | ? | at::QScheme | QScheme |
 
-Our confidently supported types are the ones in the table that have completed rows. For a limited set of use cases, we also implicitly support any literal type that is representable within 64 bits as StableIValues, as the default reinterpret_cast will succeed. You can work with StableIValue abstractions in your custom kernel for types such as c10::Device even if there is no standard defined representation of device in custom extensions. For example, a custom operator can take as argument a StableIValue device and directly pass it through to an aten operator with aoti_torch_call_dispatcher.
+Our confidently supported types are the ones in the table that have completed rows. You can rely on this subset proper ABI stability.
```

Comment:
"rely on this subset proper ABI stability." -> "rely on this subset for proper ABI stability."